test/recipes/25-test_verify.t

```diff
@@ -369,6 +369,19 @@ SKIP: {
     skip "EC is not supported or FIPS is disabled", 7
         if disabled("ec") || $no_fips;
 
+    $ENV{OPENSSL_CONF} = $provconf;
```

Comment:
Why relocate these?

Reply:
They are not dependent on the FIPS provider version. There was also a bug in that skip command below - only 3 testcases were marked as skipped but these 4 were skipped too.